starter-project-web-react/components/Blog/blogPostForm.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
one more thing we did not discuss, let us use "PascalCase" for component names and "kebab-case" for page names
Basically meaning BlogPostForm instead of blogPostForm

Reply:
done